class SaveCallback
!!!7979653.java!!!	SaveCallback(inout callback : ISaveCallback)
        mCallback = callback;
!!!7979781.java!!!	onSuccess() : void
        assertNotCalled();
        mCalled = true;
        try {
            mCallback.onSuccess();
        } catch (RemoteException e) {
            e.rethrowAsRuntimeException();
        }
!!!7979909.java!!!	onFailure(inout message : CharSequence) : void
        assertNotCalled();
        mCalled = true;
        try {
            mCallback.onFailure(message);
        } catch (RemoteException e) {
            e.rethrowAsRuntimeException();
        }
!!!7980037.java!!!	assertNotCalled() : void
        if (mCalled) {
            throw new IllegalStateException("Already called");
        }
